test/fixtures/requests/index.js

```diff
@@ -8,18 +8,30 @@ module.exports = {
   },
   prepareOrderCancellation: require('./prepare-order-cancellation'),
   preparePayment: require('./prepare-payment'),
+  preparePaymentMinAmountXRP: require('./prepare-payment-min-xrp'),
```

Comment:
Can we use the convention of namespacing by method name (here and in other fixtures index files)? e.g. `preparePayment.minAmounXRP`